zb-classifier.js

```diff
@@ -2040,12 +2040,12 @@ class ZigbeeClassifier {
 
     if (node.modelId === 'TRADFRI remote control') {
       const genScenesOutputEndpoint =
-        node.findZhaEndpointWithOutputClusterIdHex(CLUSTER_ID.GENSCENES_HEX);
-      if (genScenesOutputEndpoint) {
-        const sceneProperty =
-          this.addButtonSceneProperty(node, genScenesOutputEndpoint);
-        sceneProperty.buttonIndex = 4;
-      }
+        node.findZhaEndpointWithOutputClusterIdHex(CLUSTER_ID.GENSCENES_HEX) ||
+        '1';    // fake in endpoint 1 as version E1810 is missing the GENSCENES output cluster
```

Comment:
I don't understand how this is going to work if the output cluster doesn't actually exist.

Reply:
See the debugging I did in https://github.com/mozilla-iot/zigbee-adapter/issues/199

From what I can workout from the log fragment below, the output cluster is not reported or created at startup but does get a message later
> zigbee-adapter: handleButtonCommand: no property found for: profileId: 260 endpoint: 1 clusterId: 5

Reply:
That’s odd. @dhylands do you have any thoughts?